### PR TITLE
REL-2888: Made changes to bump PIT version to 2017A.1.1

### PR DESCRIPTION
**Pull request by sraaphorst**
Changes needed to prepare for PIT 2017A.1.1.

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala [hunk 1]
```diff
@@ -295,20 +305,6 @@ case object SemesterConverter2014BTo2015A extends SemesterConverter {
   }
   override val transformers = List(gsSubmission)
 }
-/**
- * This converter only changes the schema version but retains the semester
- */
-case object SchemaVersionConverter extends SemesterConverter {
```
Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: We shouldn't delete this. even if it is unused, it is useful when we have bugfix releases inside the same semester

**Comment by cquiroz** [comment]
Looks good to me :+1:
Great Sebastian, the PIT is yours now 😄

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala (hunk 1 above)
Comment (sraaphorst) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: Sure! I'll re-add it.

**Comment by sraaphorst** [comment]
I am filled with excitement and a mild sense of terror 😄

**Pull request merged**